ui/src/components/MenuInput.test.tsx

```diff
@@ -167,6 +167,7 @@ describe('multiple services', () => {
             // check sub menu is rendered
             expect(screen.queryByText('test-subservice1-title1')).toBeInTheDocument();
             expect(screen.queryByText('test-subservice-subTitle2')).toBeInTheDocument();
+            await new Promise((r) => setTimeout(r, 50)); // wait for animation to end
```

Comment:
was it working before without timeout? what changed?

Reply:
i replaced it with 
` await waitFor(() => screen.queryByText('test-group-title1'), { timeout: 1000 });`

why that change:
due to some animation happening after
` await userEvent.click(screen.getByText('test-group-title1'));`
some small animation is happening and the elements locally sometimes does not appear in time, which was causing  false fail of test.

Reply:
How did it work before? I don't see any changes in the PR that may affect this test.
Animation is disabled:
https://github.com/splunk/addonfactory-ucc-generator/blob/de74f1a8397f03f3dc3729e8222910e1bb919488/ui/src/components/MenuInput.test.tsx#L29

Reply:
yea there was no change for that one recently, not sure how it works for you but for me that specific test fails from time to time when running local tests on ui.
It occurs for me on all branches since the very beginning, but if thats only for me i can handle it. Usually second of third run passes that test so no problem, just annoying.